### ISSUE TITLE
__lt__ missing for NamedTuple

### ISSUE THREAD

**Issue by ytimenkov** [microsoft/pyright#1098]
**Describe the bug**
This code issues a warning: `"__lt__" is not definedPylance (reportUndefinedVariable)`

```
class TimerItem(NamedTuple):
    deadline: float
    callback: Callable

    def __lt__(self, rhs: "TimerItem") -> bool:
        return self.deadline < rhs.deadline
```

Adding another method like `__repr__` doesn't issue a warning

Sounds similar to recently fixed methods for dataclass...

I have Pylance VS Code extension v2020.10.0

**Comment by erictraut** [comment]
Thanks for the bug report.

I've confirmed that this is related to the semantic highlighting feature in pylance. It appears to have been fixed already. I'm able to repro it with last week's pylance release, but it no longer repros with the latest code.

**Comment by erictraut** [comment]
This is addressed in Pyright 1.1.79, which I just published. It will be in the next release of pylance as well.
